README.md

```diff
@@ -73,19 +77,36 @@ The GitHub App serves as the foundation for sending webhook events to App A and
 *Note: You will need one GitHub App per org. Allowing installation to "Any account" makes it difficult to change access if installed on orgs outside your control.
 
 #### **Add secrets to Azure KeyVault**
-Runner Password: `az keyvault secret set --name azure-runner-default-password --vault-name <kv-name> --value $(uuidgen)`
 
-GitHub Client Secret: `az keyvault secret set --name github-client-secret --vault-name <kv-name> --value <secret-value>`
+Runner Password:
 
-GitHub Private Key: `az keyvault secret set --name github-private-key --vault-name <kv-name> --file <location/pem> --encoding utf-8`
+```bash
+az keyvault secret set --name azure-runner-default-password --vault-name <kv-name> --value $(uuidgen)
```

Comment:
thinking toward ease of copy/pasting, would it make sense to put the <replace me> value at the end of each line, or to refer to an env var instead? maybe something like this?
```suggestion
az keyvault secret set --name azure-runner-default-password --value $(uuidgen) --vault-name $KEYVAULT_NAME
```
I'm a big fan of the change to `bash` blocks here.

Reply:
Makes sense. Would definitely be more user friendly. Do you think referencing it as an env var is clear enough? Would it make sense to add a line above setting the env var to a placeholder value wrapped by `<>`?

Reply:
I was thinking the same thing; maybe so?